sdk/src/sys/rand.rs

```diff
@@ -11,34 +11,34 @@ use crate::sys::ErrorNumber::*;
 super::fvm_syscalls! {
     module = "rand";
 
-    /// Gets 32 bytes of randomness from the ticket chain.
+    /// Gets 32 bytes of randomness from the ticket chain,
+    /// there is no lookback limit.
     ///
     /// # Arguments
     ///
     /// - `epoch` is the epoch to pull the randomness from.
     ///
     /// # Errors
     ///
-    /// | Error               | Reason                  |
-    /// |---------------------|-------------------------|
-    /// | [`LimitExceeded`]   | lookback exceeds limit. |
-    /// | [`IllegalArgument`] | invalid buffer, etc.    |
+    /// | Error               | Reason                 |
+    /// |---------------------|------------------------|
+    /// | [`IllegalArgument`] | epoch is in the future |
```

Comment:
"epoch is not a valid prior epoch number"? e.g. negatives, too

Reply:
It'll just return randomness from epoch 0. Historically it hashed it with the epoch number (giving each epoch unique randomness), now we expect that to happen in the actors themselves.

I'll document that. Also, I'll document null tipset behavior (or try to... it changes network version to network version).